src/tools/pub/publishing.md

```diff
@@ -319,13 +319,59 @@ pub.dev updates the package listing to display
 `2.0.0` as the stable version of the package.
 
 If all of the conditions at the beginning of this section are true,
-then you can ignore the following warning from `dart pub publish`: 
+then you can ignore the following warning from `dart pub publish`:
 
    *"Packages with an SDK constraint on a pre-release of the Dart SDK should
    themselves be published as a pre-release version. If this package needs Dart
    version 2.12.0-0, consider publishing the package as a pre-release
    instead."*
 
+
+## Retracting a package version {#retract}
+
+To prevent new package consumers from adopting a recently
+published version of your package, you can retract that package version
+within 7 days of publication.
+The retracted version can be restored again within 7 days of retraction.
```

Comment:
@mit-mit to clarify if the the 7-day window keeps getting extended...or not.

Reply:
@szakarias @jonasfj the question is: if a developer toggles between retracted or not, does that A) add 7 days to the window, or B) doesn't change the window; i.e. the window is always 7 days after publication ?

Reply:
The window extends so that you always have 7 days to undo the retraction. But 7 days after publishing you can't flip it to retracted again.